doc/source/management/data_at_rest_encryption.rst

```diff
@@ -585,7 +583,46 @@ on every other key rotation.
 
 This variable simultaneously turns on the encryption of files used by InnoDB for
 full text search using parallel sorting, building indexes using merge sort, and
-online DDL logs created by InnoDB for online DDL. Encryption is available for file merges used in queries and backend processes.
+online DDL logs created by InnoDB for online DDL.
+
+.. _data-at-rest-encryption.undo-tablespace:
+
+InnoDB Undo Tablespace Encryption
+================================================================================
+
+:Availability: This feature is **Experimental** quality
```

Comment:
This is an upstream feature for 8.0, and GA.

It is also possible to remove undo encryption in 8.0, by truncating the undo tablespaces.